qt-models/filtermodels.h

```diff
@@ -137,14 +50,14 @@ public
 	void stopFilterDiveSite();
 	void filterChanged(const QModelIndex &from, const QModelIndex &to, const QVector<int> &roles);
 	void setLayout(DiveTripModel::Layout layout);
-
+	void filterDataChanged(const FilterData& data);
 signals:
 	void filterFinished();
 private:
 	MultiFilterSortModel(QObject *parent = 0);
-	QList<FilterModelBase *> models;
 	struct dive_site *curr_dive_site;
 	DiveTripModel *model;
+	FilterData m_filterData;
```

Comment:
None of the other member variables have the `m_` prefix. Even though we are super-inconsistent with respect to this, at least stay consistent inside of a class ... ?

Reply:
yeah, let's call this `filterData`